cmp/grid/enums/TreeStyle.js

```diff
@@ -24,14 +24,14 @@ export const TreeStyle = Object.freeze({
 });
 
 
-export function getTreeStyleClasses(treeStyle) {
+export function getTreeStyleClasses(treeStyle, maxDepth) {
     switch (treeStyle) {
         case TreeStyle.HIGHLIGHTS:
-            return 'xh-grid--tree-style-highlights';
+            return `xh-grid--tree-style-highlights-level-${maxDepth}`;
```

Comment:
We would still want to have a base `xh-grid--tree-style-highlights` class applied to generally key off of. If we went this route in general, we would apply another class with maxDepth baked in, as an addition.